### PR TITLE
User change email

### PR DESCRIPTION
**Pull request by LomyW**

### CODE REVIEW

File: src/pages/VerifyEmailChange.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    const navigate = useNavigate();
+    const { t } = useTranslation();
+    const [searchParams] = useSearchParams();
+    const token = searchParams?.get('secret_token') || '';
+
+    const [showSuccess, setShowSuccess] = useState<boolean>(false);
+    const { sessionState } = useApollo();
+
+    const [verifyNewEmail] = useMutation(
+        gql(`mutation verifyNewEmail($token: String!) {
```
[the remark this answers is not in the document]
Comment (LomyW) [reply src/pages/VerifyEmailChange.tsx:21]: do we need a new component at all?

[the remark this answers is not in the document]
Comment (Jonasdoubleyou) [reply src/pages/VerifyEmailChange.tsx]: Not until a designer has the idea to change one page but not the other :)

[the remark this answers is not in the document]
Comment (Jonasdoubleyou) [reply src/pages/VerifyEmailChange.tsx]: different email templates

File: src/pages/ChangeEmail.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        lg: sizes['formsWidth'],
+    });
+
+    const buttonWidth = useBreakpointValue({
+        base: '100%',
+        lg: sizes['desktopbuttonWidth'],
+    });
+
+    const handleKeyPress = useCallback(
+        (e: NativeSyntheticEvent<TextInputKeyPressEventData>) => {
```
Comment (Jonasdoubleyou) [inline src/pages/ChangeEmail.tsx:39]: I find it counter intuitive that I have to hit enter for the button to activate. Had to look into the code to get that 😅

File: src/pages/Settings.tsx [hunk 3]
```diff
@@ -72,6 +72,9 @@ const Settings: React.FC = () => {
                         </Column>
                     </ProfileSettingRow>
                     <ProfileSettingRow title={t('settings.account.title')} isSpace={false}>
+                        <Column mb={tabspace}>
```
Comment (Jonasdoubleyou) [inline src/pages/Settings.tsx:75]: We currently do not show the current email anywhere. Maybe we can add it to the profile, the change email page and maybe also to the notification preferences?

File: src/pages/ChangeEmail.tsx (hunk 2 above)
Comment (Jonasdoubleyou) [reply src/pages/ChangeEmail.tsx:39]: Can't we just do `useMemo(() => isValidEmail(email), [email])` ?

**Review by Jonasdoubleyou — APPROVED** [review]

**Pull request merged**